jenkinsfile-examples/android-build-flavor-from-branch/JenkinsFile

```
@@ -15,14 +15,14 @@ node {
   echo "Building flavor ${flavor}"
 
   //build your gradle flavor, passes the current build number as a parameter to gradle
-  sh "./gradlew clean assemble${flavor}Debug -PBUILD_NUMBER=${env.BUILD_NUMBER}"
+  sh './gradlew clean assemble${flavor}Debug -PBUILD_NUMBER=${env.BUILD_NUMBER}'
 
   stage 'Stage Archive'
   //tell Jenkins to archive the apks
   archiveArtifacts artifacts: 'app/build/outputs/apk/*.apk', fingerprint: true
 
   stage 'Stage Upload To Fabric'
-  sh "./gradlew crashlyticsUploadDistribution${flavor}Debug  -PBUILD_NUMBER=${env.BUILD_NUMBER}"
+  sh './gradlew crashlyticsUploadDistribution${flavor}Debug  -PBUILD_NUMBER=${env.BUILD_NUMBER}'
```

Won't work. This probably will:
```suggestion
  sh './gradlew crashlyticsUploadDistribution${flavor}Debug  -PBUILD_NUMBER=${BUILD_NUMBER}'
```